app/(main)/filters.tsx

```diff
@@ -3,9 +3,9 @@ import {
   FixedProductMarketMaker_OrderBy,
   OrderDirection,
 } from '@/queries/omen';
-import { AI_AGENTS_ALLOWLIST } from '../constants';
+import { AI_AGENTS_ALLOWLIST } from '../../constants';
 import { _24HoursInSeconds, nowTimestamp } from '@/utils/time';
-import {WXDAI, SDAI} from '@/constants/index';
+import { WXDAI, SDAI } from '@/constants/index';
```

Comment:
Looks like we can get `AI_AGENTS_ALLOWLIST`, `WXDAI`, `SDAI` from `@/constants`

Reply:
true and we can remove `/index`